Keyboard.md

```diff
@@ -0,0 +1,25 @@
+# Keyboard Driver
+
+## IRQ and IO-Apic
+
+* The Keyboard IRQ is the number 1. This is mapped to the IO Redirection Table 1, at offset 12h and 13h (called IOREDTBL1). 
```

Comment:
this is a good example, but perhaps its worth being more general when describing this:
Irq1 will be pin 1 on the IO APIC, which is controlled by entry 1 in the redirect table. This register is accessed as 2x 32bit registers, where the register number is as follows: 0x10 + (irq_number * 2). In this case that means the lower 32 bits of irq 1 are in register 0x12, with the upper 32 bits in 0x13.